events/open-standup.md

```diff
@@ -5,32 +5,25 @@ description: How do we do whole-team standup at Artsy?
 
 # Dev Team Standup at Artsy
 
-We've been doing engineering standups for as long as there has been an engineering team. The formats have changed
-over time as the size and scope of Artsy has changed. We're now on our fourth iteration of this process.
+We've been doing engineering standups for as long as there has been an engineering team. The format has changed over time as the size and scope of Artsy has changed. We're now on our fourth iteration of this process.
 
-A standup is ideally ran by two people, one person talking, the other taking notes. Decide amongst yourselves which
-of you will do the before standup stuff.
-
-**Note**: We document our standup process to the degree that _anyone_ on our team can run the standup because we
```

Comment:
Yeah, I think it makes sense to remove this now that standups are a formal responsibility for the on-call engineers 👍

Reply:
Yeah, it was a good note, but things change. ❤️